project/plugins.sbt

```diff
@@ -1,2 +1,7 @@
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.7.5")
 addSbtPlugin("com.frugalmechanic" % "fm-sbt-s3-resolver" % "0.19.0")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
+libraryDependencies += "com.sun.activation" % "javax.activation" % "1.2.0"
+addSbtPlugin("ohnosequences" % "sbt-github-release" % "0.7.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
```

Comment:
Are all of these plugins necessary, or were you just testing some of them out?

Reply:
They're all needed :smile: `sbt-release` does the versioning and co-ordinates the other changes. `sbt-github-release` does the actual release to github. `javax.activation` is needed by `sbt-github-release` because the maintainer hasn't updated the plugin for the latest versions of java where this library is no longer included and the `sbt-build-info` one generates the `BuildInfo` class that we get the current version from.